### DISCUSSION TITLE
Help Please

### DISCUSSION THREAD

**Discussion by egemenkorkmaz** [mckaywrigley/chatbot-ui#1304] [Help]
I am creating a membership but this error is always present

![Ekran Resmi 2024-01-29 00 42 39](https://github.com/mckaywrigley/chatbot-ui/assets/19240650/d911e9ff-49da-4151-9260-560296252233)
![Ekran Resmi 2024-01-29 00 41 41](https://github.com/mckaywrigley/chatbot-ui/assets/19240650/74269543-a128-45fc-afdd-27582de9dd77)

**Comment by mhalavanja** [comment]
Did you make it work? I think I had errors like this before I did the migrations.

**Reply by mhalavanja** [reply]
Also, I remember that reseting docker and supabase helped with something similar as well.

**Reply by egemenkorkmaz** [reply]
Yes, I reset the computer and restarted it and it worked.
